lib/ansible/module_utils/remote_management/dellemc/dellemc_idrac.py

```diff
@@ -0,0 +1,97 @@
+# _*_ coding: utf-8 _*_
+
+#
+# Dell EMC OpenManage Ansible Modules
+# Version 1.0
+# Copyright (C) 2018 Dell Inc.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# All rights reserved. Dell, EMC, and other trademarks are trademarks of Dell Inc. or its subsidiaries.
+# Other trademarks may be trademarks of their respective owners.
+#
+
+from __future__ import (absolute_import, division,
+                        print_function)
+import tempfile
+import os
+__metaclass__ = type
```

Comment:
Usually PEP8 recommends to place "dunder" names before doing anything else: https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names.

Could you please move this line right after future import? TIA.

Reply:
removed the extra imports and moved the __metaclass__ next to future import